app/src/main/res/layout/fragment_sign_out.xml

```diff
@@ -1,56 +1,36 @@
-    tools:context = ".ui.authentication.SignOutFragment">
+    android:gravity="center_horizontal"
+    tools:context=".ui.authentication.SignOutFragment"
+    android:theme="@style/Theme.Tarjetakuna.NoActionBar">
 
     <TextView
-        android:id="@+id/greetingMessage"
-        android:layout_width="231dp"
-        android:layout_height="101dp"
+        android:id="@+id/signOut_greeting_text"
+        style="@style/TitleStyle"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_margin="@dimen/text_margin"
         android:gravity="center|center_horizontal"
         android:text="@string/hello"
-        android:textSize="20sp"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        android:layout_weight="1" />
 
     <Button
-        android:id="@+id/signOutButton"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="@string/sign_out"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.498"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="0.73" />
+        android:id="@+id/signOut_sign_out_button"
+        style="@style/ButtonStyle"
+        android:text="@string/sign_out" />
 
     <Button
-        android:id="@+id/button_back_home"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="@string/go_back_to_home_page"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.498"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="0.802" />
+        android:id="@+id/signOut_manage_collection_button"
+        style="@style/ButtonStyle"
+        android:text="@string/manage_collection_button" />
 
     <Button
-        android:id="@+id/manage_collection_button"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginBottom="52dp"
-        android:text="@string/manage_collection_button"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.5"
-        app:layout_constraintStart_toStartOf="parent" />
-
-</androidx.constraintlayout.widget.ConstraintLayout>
+        android:id="@+id/signOut_home_button"
+        style="@style/HomeButtonStyle"
+        android:text="@string/go_back_to_home_page"
+        tools:ignore="MissingConstraints" />
```

Comment:
Should we really use "ignore missingConstraints" here?

Reply:
the style HomeButtonStyle imports the constraints (as our home buttons are all in the same place it's efficient to just define that elsewhere), it just complains if you dont explicity write contraints within the layout